README.md

```diff
@@ -250,7 +278,23 @@ will take precedence.
 
 Hyrax is available under [the Apache 2.0 license](LICENSE.md).
 
-# Contributing
+# Quick Start for Hyrax development and contributing code back
+
+The following steps need to be done in order to create a test app for Hyrax development.  This is a quick list with links to the details.
+
+1. [Install prerequisite software](#install-prerequisite-software) - Follow all instructions carefully.
+1. Clone Hyrax code from github
+1. Remove existing test app with `rake engine_cart:clean` (Not required after initial clone. Use when your code updates require the test app to be regenerated.)
+1. Create the test app with `rake engine_cart:generate`
+1. If using rubyracer for java runtime, uncomment in `.internal_test_app/Gemfile` and bundle install.  (Not needed if using nodejs.) ([more info](#javascript-runtime))
+1. [Start servers](#start-servers) with `rake hydra:server`  (e.g. solr, fedora, rails) - Stop with <CTRL><C>
```

Comment:
`<CTRL><C>` isn't showing up for me when I view this file rendered by Github.  Maybe just use `Ctrl-C` or `^C`?